csrankings.csv

```diff
@@ -18777,6 +18779,7 @@ Tobias Scheffer,University of Potsdam,https://www.uni-potsdam.de/en/cs-ml/staff/
 Tobias Schreck,Graz University of Technology,https://www.tugraz.at/institute/cgv/schreck,9cJsSIIAAAAJ
 Tobias Weinzierl,Durham University,https://www.dur.ac.uk/computer.science/staff/profile/?id=12329,pRrvub4AAAAJ
 Tobias Wrigstad,Uppsala University,http://wrigstad.com,EfO5KRQAAAAJ
+Tobin Isaac,Georgia Institute of Technology,https://triquadtethex.org,SpDPpqgAAAAJ

```

Comment:
Problem with https on this site.

Reply:
Sorry I will ask him to fix his personal web site. But here is his official GT CSE Faculty page: https://www.cse.gatech.edu/people/tobin-isaac

Reply:
Sorry it seems like http://triquadtethex.org (non-secure) link works. let me know if you want me resubmit merge request, or if you merge and fix together.

Reply:
If you can update, that'd be easiest.

Reply:
fixed